src/utils/enums.py

```diff
@@ -0,0 +1,10 @@
+from foobar.exceptions import InvalidTransition
+
+
+def validate_transition(enum, from_state, to_state):
+    if not hasattr(enum, '_transitions'):
+        raise InvalidTransition
```

Comment:
It could be a good idea to provide an exception message when throwing this (no transitions found) and the exception below (transition A -> B not allowed). Just so one does not need to dig in the code to see why the exception was raised.